modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx

```diff
@@ -24,6 +24,7 @@ const languages = Liferay.Language.available;
 const languageLabels = Object.values(languages).map((language) => {
 	return {label: language};
 });
+const defaultLanguage = Object.values(languageLabels[0]).toString();
```

Comment:
Hi @vngoliveira, you don't need to declare this `defaultLanguage`, you can just use the `defaultLanguageId` that already exists in this file :)

Reply:
Hello, @naraandrade! I declared this `defaultLanguage` because when I used the `defaultLanguageId` the format was different than the others options in the dropdown. How I can use the `defaultLanguageId` with the default format others?